alembic/versions/b74cc308b1ec_move_responses_to_db.py

```diff
@@ -0,0 +1,83 @@
+revision = 'b74cc308b1ec'
+down_revision = '66d78d1a29ee'
+branch_labels = None
+depends_on = None
+
+import alembic
+import sqlalchemy
+import json
+from collections import defaultdict
+from www.commands import build_response_dict
```

Comment:
This import doesn't work. I tried to add the project root to the import path but for some reason that breaks Alembic.